samps/contests/ccs1/config/contest.yaml

```diff
@@ -13,6 +13,9 @@ output-private-score-dir: super_secret
 output-public-score-dir: public_html_dir
 
 auto-stop-clock-at-end: true
+memory-limit-in-Meg: 8086
+
+sandbox: 'linux_sandbox --verbose'

```

Comment:
Does the above line have the effect of specifying that the "ccs1" sample contest requires a linux sandbox?  What happens if someone tries to load that sample contest under Windows?   (If it's going to fail under Windows, I think the sample should be split off into a new sample contest with a different name -- something that includes in its name an indication that it is requiring Linux and using a sandbox.)

Reply:
I think that is left-over from when I copied over your branch.   TBH I didn't even look in the samps folder.